portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -255,14 +255,46 @@ definition {
 	}
 
 	@description = "Verify that a translation can be previewed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBePreviewed {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133520 CanBePreviewed pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		Form.editFieldLabelMetal(fieldValue = "Text ES");
+
+		Form.save();
+
+		FormsAdminNavigator.gotoPreviewForm();
+
+		Navigator.selectSpecificWindow(windowTitle = "Form - Forms");
 
+		FormPortletConfiguration.viewTranslationLanguage(symbolName = "en-US");
+
+		FormFields.viewLabel(
+			fieldLabel = "Text",
+			fieldName = "Text");
+
+		FormPortletConfiguration.changeLanguageTo(changeLanguageTo = "en-US");
+
+		FormPortletConfiguration.changeLanguageTo(changeLanguageTo = "español-España");
```

Comment:
@wesleyroberts, apparently, the macro `Form.selectTranslation` can't help you here, right?

In addition, I realized that the following pattern isn't being used, is there any special reason for it? If not, can you please use the pattern?

```
var key_changeLanguageTo = "${changeLanguageTo}";
```

Reply:
I can use it, I didn't make the macro, just take advantage of a co-worker

Reply:
Changes made

Reply:
@wesleyroberts, the pattern is still different. :disappointed: Please see my previous comment.

Reply:
Sorry for the mistakes, the changes were made